### PR TITLE
Adding 'Updating Terraform Enterprise License' section to Application Administration section

### PR DESCRIPTION
**Pull request by goldendemise**
## Labels

- [ ] inaccuracy
- [ ] clarification
- [x] new docs
- [ ] cosmetic bug - fixing broken text or markup
- [ ] enhancement - changing the website's behavior/layout
- [ ] api - requires an update to the [changelog](https://www.terraform.io/docs/cloud/api/changelog.html)

### CODE REVIEW

File: content/source/docs/enterprise/admin/update-tfe-license.html.md
Comment (justinretzolk) [inline content/source/docs/enterprise/admin/update-tfe-license.html.md:58]: We should wrap this in a code fence

````suggestion
    ```shell
        $ replicatedctl license inspect
        [
        {
           "ID": "670bd320b95245325d80be703330f9b3",
           "Assignee": "Test Customer Name",
           "LegacyChannelName": "Stable",
           "ChannelID": "",
           "Channels": [],
           "ExpirationTime": "2021-06-30T00:00:00Z",
           "ExpirationPolicy": "ignore",
           "IsExpired": false,
           "IsActivationRequired": false,
           "ActivationEmail": "",
           "Fields": [
               {
                   "FieldName": "max_hosts",
                   "FieldTitle": "Maximum Number of Hosts",
                   "FieldType": "Integer",
                   "Value": "9999"
               },
               {
                   "FieldName": "min_hosts",
                   "FieldTitle": "Minimum Number of Hosts",
                   "FieldType": "Integer",
                   "Value": "1"
               }
           ]
        }
        ]
    ```

````

File: content/source/docs/enterprise/admin/update-tfe-license.html.md
Comment (justinretzolk) [inline content/source/docs/enterprise/admin/update-tfe-license.html.md:119]: Wrapped in code fence + broke the long line up a bit for readability.

````suggestion
        ```shell
        $ replicatedctl license-load \
            --airgap-package /path/to/bundle.airgap < /path/to/license.rli
        ```
````

File: content/source/docs/enterprise/admin/update-tfe-license.html.md
Comment (justinretzolk) [inline content/source/docs/enterprise/admin/update-tfe-license.html.md:131]: Wrapped in code fence.

````suggestion
         ```plaintext
         WARN 2021-02-22T01:40:00+00:00 tasks/app_tasksteps.go:113 Airgap license on disk does not match installed license
         ```
````

File: content/source/docs/enterprise/admin/update-tfe-license.html.md
Comment (justinretzolk) [inline content/source/docs/enterprise/admin/update-tfe-license.html.md:139]: Wrapped in code fence

````suggestion
    ```plaintext
    Unable to sync license: Error: Unsuccessful HTTP response
    ```
````

File: content/source/docs/enterprise/admin/update-tfe-license.html.md
Comment (justinretzolk) [inline content/source/docs/enterprise/admin/update-tfe-license.html.md:153]: Wrapped in code fence.

````suggestion
    ```plaintext
    installed app release (325b33bf0ad539c994644423128cad5e:502) does not match the airgap package
    ```
````

**Review by justinretzolk — CHANGES_REQUESTED** [review]
The content of the new page looks good overall, save for a few things that needed to be wrapped in a code fence. I'll defer to @nfagerlund for the other bits of the PR that relate to organizing the page within the site, etc.

File: content/source/docs/enterprise/admin/update-tfe-license.html.md
Comment (laurapacilio) [inline content/source/docs/enterprise/admin/update-tfe-license.html.md:165]:
```suggestion
The airgap download page displays the versions of Terraform Enterprise and their SHA256 checksum values. Check these against the version of the airgap package in your local environment.
```

**Review by laurapacilio — APPROVED** [review]
Hi,

I'm the new technical writer - nice to meet you! :-) I can't review for accuracy, but I did leave a few comments on style and flow. See what you think - thank you!

File: content/source/docs/enterprise/admin/update-tfe-license.html.md
Comment (laurapacilio) [inline content/source/docs/enterprise/admin/update-tfe-license.html.md:5]:
```suggestion
```

**Comment by laurapacilio** [comment]
@goldendemise Did you still want to get this merged in? I had a couple of suggestions for you to review, and then beyond that we'll be good to go!

File: content/source/docs/enterprise/admin/update-tfe-license.html.md
Comment (justinretzolk) [inline content/source/docs/enterprise/admin/update-tfe-license.html.md:26]: You'll want a line break here to make sure formatting is preserved

```suggestion
2.  Use the `replicatedctl license inspect` command to inspect the license information. The `ExpirationTime` value in the output indicates the expiration date/time in UTC.

```

**Review by justinretzolk — APPROVED** [review]
Left one minor comment, but approving so as to not be the holdup. I agree with the other suggestions made as well, so leaving them as is.

File: content/source/docs/cloud/run/api.html.md
Comment (laurapacilio) [inline content/source/docs/cloud/run/api.html.md:60]:
```suggestion
The [configuration version API](../api/configuration-versions.html) requires an uploaded `tar.gz` file to use the configuration version for a run, so you must package the directory containing the Terraform configuration into a `tar.gz` file.
```

File: content/source/docs/enterprise/admin/update-tfe-license.html.md
Comment (laurapacilio) [inline content/source/docs/enterprise/admin/update-tfe-license.html.md:151]:
```suggestion
    Terraform Enterprise requires network communication to the Replicated endpoints, as noted in the [network requirements](https://www.terraform.io/docs/enterprise/before-installing/network-requirements.html) guideline. During the license sync operation, the Replicated installer attempts to contact `api.replicated.com` to retrieve the license information. In the online installation type, you might see the above error if the network infrastructure changed after you installed Terraform Enterprise. The specific IP addresses of Replicated services for the Terraform Enterprise are available in [Replicated’s GitHub repository](https://github.com/replicatedhq/ips/blob/master/ip_addresses.json).
```

File: content/source/docs/enterprise/admin/update-tfe-license.html.md
Comment (laurapacilio) [inline content/source/docs/enterprise/admin/update-tfe-license.html.md:155]:
```suggestion
    This may occur with an Airgap installation for a variety of reasons. One example is when a trial license is associated with a different customer name from the subsequent paid license.  Please contact [HashiCorp Support](https://www.hashicorp.com/technical-support-services-and-policies) for assistance. In your request, include the output from running`replicatedctl license inspect` on the Terraform Enterprise host machine.
```

**Review by laurapacilio — APPROVED** [review]
Two more tiny nits :-) Thank you so much. After you merge these two things, feel free to merge this PR 🚀

**Pull request merged**
